docs/source/learning_materials.rst

```diff
@@ -0,0 +1,12 @@
+
+.. _learning_materials:
+
+Learning Materials
+=======================================
+
+**git-machete** is a handy tool for managing your git repositories and to make it easier to use author of the tool Paweł Lipski has prepared series of articles on various git-machete related topics. Check them out for more information on how to use git-machete or how to manage your own programming project using Docker and CI services:
+
+    * `Make your way through the git (rebase) jungle with Git Machete <https://medium.com/virtuslab/make-your-way-through-the-git-rebase-jungle-with-git-machete-e2ed4dbacd02>`_;
+    * `Git Machete Strikes again! <https://medium.com/virtuslab/git-machete-strikes-again-traverse-the-git-rebase-jungle-even-faster-with-v2-0-f43ebaf8abb0>`_;
```

Comment:
Nit: use the full(er) title to hint about the scope: `git machete strikes again! Traverse the git (rebase) jungle even faster` (that `traverse` is covered there)